src/main/java/mods/railcraft/common/blocks/machine/alpha/TileSteamOven.java

```diff
@@ -154,13 +156,25 @@ public int getCookTime() {
         return -1;
     }
 
+    @Override
+    @Nonnull
     public EnumFacing getFacing() {
         TileSteamOven masterOven = (TileSteamOven) getMasterBlock();
         if (masterOven != null)
             return masterOven.facing;
         return facing;
     }
 
+    @Override
+    public void setFacing(@Nonnull EnumFacing facing) {
+        TileSteamOven masterOven = (TileSteamOven) getMasterBlock();
+        if (masterOven != null) {
+            masterOven.facing = facing;
+        }
+        this.facing = facing;
+        WorldPlugin.setBlockState(worldObj, getPos(), getBlockState().withProperty(BlockMachineAlpha.FRONT, facing));
```

Comment:
But it does nothing with the meta. It just update the facing of the tile being called.